lang/dart.md

```diff
@@ -0,0 +1,188 @@
+---
+lang: Dart
+syntax: dart
+title: Dart is Smart Language!
+author: chandan0000
+---
+
+```dart
+void main(){
+    print("hello world");
+}
+```
+
+## What is Dart?
```

Comment:
As @vsnthdev said, we should add an extra line here. 🙂
```suggestion
## What is Dart?

```

Reply:
@Panquesito7: Yes! Thanks to your catchy evil eye! : )